examples/setall.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import sys
+from sys import argv
+import blinkt
+
+if len(argv)<2 or len(argv)>5:
```

Comment:
If you run `./setall.py 0` or `./setall.py 0 0` then the script will fail to print the usage instructions.

Should probably be:

```python
if len(argv) < 4 or len(argv) > 5:
```

Reply:
Done